From bc746413e07c504535873cf860a8f2c38862896e Mon Sep 17 00:00:00 2001
From: MilhouseVH <milhouseVH.github@nmacleod.com>
Date: Thu, 1 Feb 2018 08:15:27 +0000
Subject: [PATCH] Fix build failure with recent ax_prog_doxygen.m4

Recent autotool-archives doxygen macros breaks the current build,
so this patch updates to the method of the latest autoconf-archive version.

The required autoconf-archive macro has been committed to their repository
in November 2015, so anything later should work.

Ref: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=837020
---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 33813b0..f92e741 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,5 @@
-include $(top_srcdir)/doxygen-include.am
+# include Doxygen rules (requires autoconf-archive >2016-03-20)
+@DX_RULES@
 
 ACLOCAL_AMFLAGS = -I m4
 
-- 
2.14.1

